docs/api/cases/cases-api-get-case.asciidoc

```diff
@@ -0,0 +1,101 @@
+[[cases-api-get-case]]
+== Get case API
+++++
+<titleabbrev>Get case</titleabbrev>
+++++
+
+Returns a specified case.
+
+=== Request
+
+`GET <kibana host>:<port>/api/cases/<case ID>`
+
+`GET <kibana host>:<port>/s/<space_id>/api/cases/<case ID>`
+
+=== Prerequisite
+
+You must have `read` privileges for the *Cases* feature in the *Management*,
+*{observability}*, or *Security* section of the
+<<kibana-feature-privileges,{kib} feature privileges>>, depending on the
+`owner` of the cases you're seeking.
+
+=== Path parameters
+
+`<case_id>`::
+(Required, string) An identifier for the case to retrieve. Use 
+<<cases-api-find-cases>> to retrieve case IDs.
+
+`<space_id>`::
+(Optional, string) An identifier for the space. If it is not specified, the
+default space is used.
+
+=== Query parameters
+
+`includeComments`::
+(Optional, boolean) Determines whether case comments are returned. Defaults to 
+`true`.
+
+ deprecated:[8.0.0]
```

Comment:
I think it is missing the deprecated warning note as here: https://www.elastic.co/guide/en/security/current/cases-api-get-case.html. And it is from 8.1. Sorry, my mistake 🙂

Reply:
@cnasikas I used the standard deprecation block here as https://github.com/elastic/docs#additions-and-deprecations suggests. I fixed the version number in the warning note.